starterkits/basic/packages/primary-components/atoms/button/_a-button.css

```diff
@@ -1,11 +1,11 @@
 %a-button {
   @apply py-2 px-4 rounded bg-green text-base text-white outline-none border-0;
 
-  &:hover {
+  &--hover {
     @apply bg-green-light;
   }
 
-  &:focus {
+  &--focus {
```

Comment:
it changes toolbar
[![Image from Gyazo](https://i.gyazo.com/d95b59cdca7ae3907d3961f493b06689.png)](https://gyazo.com/d95b59cdca7ae3907d3961f493b06689)

also not really sure about default look
[![Image from Gyazo](https://i.gyazo.com/4d22984482e4a34481c53bec76bbed95.png)](https://gyazo.com/4d22984482e4a34481c53bec76bbed95)

Reply:
also affects logo and title
[![Image from Gyazo](https://i.gyazo.com/927a963f1039b334826d8037cf308386.gif)](https://gyazo.com/927a963f1039b334826d8037cf308386)

Reply:
i know, but is it really problem here if to think about accessibility?

Reply:
removed this effect. Added default text-underline: none on hover/focus + default outline